code_share/code_share/settings.py

```diff
@@ -13,7 +14,7 @@
 SECRET_KEY = '6*yy_^z%min*=(w9rjxj&f*fgl6daq3j)_52my8i)u5hp)9g6#'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = config('DEBUG'),
```

Comment:
The config will be stored in project root in .env file and since it may contain the confidencial data so we should not hardcode it or even upload the .env file to github